src/main/resources/api/service_openapi.yaml

```diff
@@ -138,6 +139,10 @@ components:
           type: object
           additionalProperties:
             type: string
+        creation:
+          description: The time when the resource is created.
+          type: string
+          format: 'date-time'
```

Comment:
did we change our minds about figuring this out in CRL with Instant.now()?

Reply:
ah, right. I was concerned client sets header randomly, and there are various date related field in header. But I think I l know how I can get this.